SubmissionExample.ipynb

```diff
@@ -364,86 +354,9 @@
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/Server/performance/run_4/results.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/Server/performance/run_5/mlperf_log_accuracy.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/Server/performance/run_5/mlperf_log_summary.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/Server/performance/run_5/mlperf_log_detail.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/Server/performance/run_5/results.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/accuracy/mlperf_log_accuracy.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/accuracy/mlperf_log_summary.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/accuracy/mlperf_log_detail.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/accuracy/results.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/accuracy/accuracy.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/performance/run_1/mlperf_log_accuracy.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/performance/run_1/mlperf_log_summary.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/performance/run_1/mlperf_log_detail.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/SingleStream/performance/run_1/results.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/accuracy/mlperf_log_accuracy.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/accuracy/mlperf_log_summary.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/accuracy/mlperf_log_detail.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/accuracy/results.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/accuracy/accuracy.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/performance/run_1/mlperf_log_accuracy.json\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/performance/run_1/mlperf_log_summary.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/performance/run_1/mlperf_log_detail.txt\r\n",
-      "/tmp/mlperf-submission/closed/mlperf-org/results/tf-gpu/mobilenet/MultiStream/performance/run_1/results.json\r\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
it might be good to include the output so the reader can see the results without running it.

Reply:
I see.  When the prior two comments are resolved, I will update and re-run for output.